### ISSUE TITLE
set manualSeed invalid

### ISSUE THREAD

**Issue by ACbccc** [machine-perception-robotics-group/attention_branch_network#9]
Hi:
I tried your code on CIFAR100 and got competitive performance. But when I used --manualSeed=10 to set the seed and used --epochs 1 to train, each time the model gives me a different result. I'm so puzzled. What can I do to make sure when I set manualSeed, I can get the same results? Thank you.

**Comment by Hiroshi-Fukui** [comment]
Hi.
Sorry, I don't know the behavior when changes '--manualSeed' because I didn't try it.
Please ask the issue to the [base code](https://github.com/bearpaw/pytorch-classification) of ABN.
